File: URI/C/Experiencias.c
#include <stdio.h>

int main()
{
    int n;
    int coelhos=0, ratos=0, sapos=0, cobaias=0, aux;
    float percentual;
    char identificador;

    scanf("%d", &n);
    int i=0;
    while(i<n)
    {
        scanf("%d %c", &aux, &identificador);
        getchar();

        if (identificador == 'C') coelhos+=aux;
        else if (identificador == 'R') ratos+=aux;
        else sapos+=aux;
        
        cobaias+=aux;
        i++;
    }
    printf("Total: %d cobaias\n", cobaias);
    printf("Total de coelhos: %d\n", coelhos);
    printf("Total de ratos: %d\n", ratos);
    printf("Total de sapos: %d\n", sapos);

    printf("Percentual de coelhos: %.2f %%\n", (float)coelhos/(float)cobaias *100);
    printf("Percentual de ratos: %.2f %%\n", (float)ratos/(float)cobaias *100);
    printf("Percentual de sapos: %.2f %%\n", (float)sapos/(float)cobaias *100);
    
    return 0;
}

File: URI/Python/Experiencias.py
n = int(input())
coelhos = 0
ratos = 0
sapos = 0

for i in range(n):
    inf = [x for x in input().split()]
    qte = int(inf[0])
    if inf[1] == 'C':
        coelhos+=int(qte)
    elif inf[1] == 'R':
        ratos+=qte
    else:
        sapos+=qte

total = coelhos+sapos+ratos

print("Total: {} cobaias".format(total))
print("Total de coelhos: {}".format(coelhos))
print("Total de ratos: {}".format(ratos))
print("Total de sapos: {}".format(sapos))

print("Percentual de coelhos: {:.2f} %".format(100*coelhos/total))
print("Percentual de ratos: {:.2f} %".format(100*ratos/total))
print("Percentual de sapos: {:.2f} %".format(100*sapos/total))
